google/cloud/spanner/testing/random_database_name.cc

```diff
@@ -13,17 +13,31 @@
 // limitations under the License.
 
 #include "google/cloud/spanner/testing/random_database_name.h"
+#include "google/cloud/internal/format_time_point.h"
+#include <ctime>
 
 namespace google {
 namespace cloud {
 namespace spanner_testing {
 inline namespace SPANNER_CLIENT_NS {
+
+std::string RandomDatabasePrefixRegex() {
+  return R"re("^db-\d{4}-\d{2}-\d{2})re";
```

Comment:
Perhaps this should have the dash on the end too (to match `RandomDatabasePrefix()`).

Reply:
Thanks, done.